File: mediapicker/src/main/java/vn/tungdx/mediapicker/MediaSelectedListener.java
package vn.tungdx.mediapicker;

import java.util.List;

/**
 * @author TUNGDX
 */

/**
 * Listener for select media item.
 *
 */
public interface MediaSelectedListener {
    public void onHasNoSelected();

    public void onHasSelected(List<MediaItem> mediaSelectedList);
}

File: mediapicker/src/main/java/vn/tungdx/mediapicker/MediaSelectedListener.kt
package vn.tungdx.mediapicker

/**
 * @author TUNGDX
 */

/**
 * Listener for select media item.
 *
 */
interface MediaSelectedListener {

    fun onHasNoSelected()

    fun onHasSelected(mediaSelectedList: List<MediaItem>)
}
